### PR TITLE
Created custom dropdown unit tests

### PR DESCRIPTION
**Pull request by chalettu**
Just unit tests.  No UX changes.

### CODE REVIEW

**Comment by chalettu** [comment]
@miq-bot add_label housekeeping
@miq-bot add_label test
@miq-bot add_label fine/no

File: client/app/shared/custom-dropdown/custom-dropdown.component.spec.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+describe('Custom Dropdown component - ', function () {
```
Comment (AllenBW) [inline client/app/shared/custom-dropdown/custom-dropdown.component.spec.js]: don't hate me for knit picking, convention throughout tests so far has been `Component: component-name`

File: client/app/shared/custom-dropdown/custom-dropdown.component.spec.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        updateSpy = sinon.stub(ctrl, 'onUpdate').returns(true);
+    });
+
+    it('is defined', function () {
+        expect(ctrl).to.be.defined;
+    });
+
+    it('should handle changes', () => {
+        ctrl.$onChanges();
+        // { $changes: ["item1", "item2"], $config: { test: "test" } }
```
Comment (AllenBW) [inline client/app/shared/custom-dropdown/custom-dropdown.component.spec.js]: eeeeek comment alert! <3

File: client/app/shared/custom-dropdown/custom-dropdown.component.spec.js (hunk 1 above)
Comment (chalettu) [inline client/app/shared/custom-dropdown/custom-dropdown.component.spec.js]: Updated test name

**Pull request merged**